src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx

```diff
@@ -78,8 +79,10 @@ import { FailedImportConflict, RetryDecision } from '../../../lib/resolve_import
 import { OverwriteModal } from './overwrite_modal';
 import { ImportModeControl, ImportMode } from './import_mode_control';
 import { ImportSummary } from './import_summary';
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import { DataSourcePicker } from '../../../../../data_source_management/public/components/data_source_picker/data_source_picker.js';
```

Comment:
nit: consider adding index file in `data_source_management/public/components/data_source_picker` and export from there

Reply:
+1

Reply:
Changed component name to cluster selector and also moved export to top level in new commit